opencensus/trace/status.py

```diff
@@ -39,17 +39,31 @@ class Status(object):
                     See: https://cloud.google.com/trace/docs/reference/v2/
                          rest/v2/Status#FIELDS.details
     """
-    def __init__(self, code, message, details=None):
-        self.code = code
-        self.message = message
+    def __init__(self, code, message=None, details=None):
+        self._code = code
+        self._message = message
```

Comment:
The changes to add `set_status` look good, but this looks like a breaking change for the sake of matching the names in a different spec.